modules/getting-started-openshift-common-terms.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-overview.adoc
+
+:_content-type: REFERENCE
+[id="getting-started-openshift-common-terms_{context}"]
+= Common Terms
+
+The following are common terms that are used within the Getting Started. These terms help you orient yourself with the content and other parts of the documentation.
+
+Kubernetes::
+Kubernetes is an open source container orchestration engine for automating deployment, scaling, and management of containerized applications.
+
+Containers::
+Containers are application instances and components that run in OCI-compliant containers on the worker nodes. A container is the runtime of an Open Container Initiative (OCI)-compliant image. An image is a binary application. A worker node can run many containers. A node capacity is related to memory and CPU capabilities of the underlying resources whether they are cloud, hardware, or virtualized.
```

Comment:
You use OCI alone, and then spell it out in the second sentence. Suggest reversing that